styles/main.css

```diff
@@ -167,43 +167,37 @@ div .back-to-top:hover {
 
 main {
     display: grid;
-    grid-template-columns: 50% 50%;
+    grid-template-columns: 1fr 1fr;
     column-gap: 2em ;
 }
- .bio {
-     grid-row: 1 / 2; grid-column: 1 / span 2;
+.bio {
+    grid-column: 1 / span 2;
 }
 .glossary {
-    grid-row: 1 / 2; grid-column: 1 / span 2;
+    grid-column: 1 / span 2;
 }
 
 #useful-links {
-    grid-row: 1 / 2; grid-column: 1 / span 2;
+    grid-column: 1 / span 2;
 }
 
 .gallery {
-    grid-row: 1 / 2; grid-column: 1 / 3;
-    max-width:100%;
+    grid-column: 1 / 3;
+    width:100%;
 }
 .gallery-thumbnails {
     display:grid;
+    width: 100%;
     grid-template-columns: 25% 25% 25% 25%;
 }
 
-.gallery-thumbnails  .row-span-2{
-    width:300px;
-    height:600px;
-}
-.gallery-thumbnails  .row-span-2.col-span-2 {
-    width:600px;
-    height:600px;
-}
-.gallery-thumbnails a {
+.gallery-thumbnails img {
     display:block;
+    width: 100%;
 }
 .row-span-2 {
-    grid-row: 1 / span 2;
+    grid-row: span 2;

```

Comment:
In this case, you don't want to specify grid-row-start because if you do it forces the grid item to ALWAYS start at line 1 in the grid, but that's not what you want.  You just want the item to span 2 rows from whichever its natural position is in the grid.